src/utils.ts

```diff
@@ -70,3 +70,16 @@ export const convertToUnixPath = (path: string): string => {
 
     return path.replace(/\\/g, '/');
 };
+
+export const getWorkspacePath = (logger: winston.Logger): string | void => {
+    if(vscode.workspace) {
+        if(vscode.workspace.workspaceFolders) {
+            return vscode.workspace.workspaceFolders[0].uri.fsPath;
```

Comment:
Is the first folder is the active one?

Reply:
The first folder will give the absolute path to the root of the workspace opened in VSCode: https://code.visualstudio.com/api/references/vscode-api#WorkspaceFolder 
 I had to use that since rootPath is deprecated. https://code.visualstudio.com/api/references/vscode-api#workspace